cloud/provider.go

```diff
@@ -203,6 +204,19 @@ func CustomPricesEnabled(p Provider) bool {
 	return config.CustomPricesEnabled == "true"
 }
 
+func NewCrossClusterProvider(ctype string, overrideConfigPath string, cache clustercache.ClusterCache) (Provider, error) {
+	if ctype == "aws" {
```

Comment:
I'm confused about how we're using this bidirectionally (AWS CCP for GCP Provider and GCP CCP for AWS Provider) but `ctype` is always `"aws"`. Shouldn't there be an option for a `ctype == "gcp"`?

Reply:
Yes, added.